timelock-server/build.gradle

```diff
@@ -54,5 +57,9 @@ dependencies {
     testCompile group: 'com.squareup.okhttp3', name: 'okhttp'
 
     integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
+    integTestCompile group: 'com.palantir.remoting3', name: 'http2-agent'
```

Comment:
I'm considering just pulling the file in rather than maintaining the extra small remoting3 dependency.